package.json

```diff
@@ -18,99 +20,89 @@
-    "linkifyjs": "^4.3.2",
+    "linkifyjs": "^4.1.1",
     "lodash.debounce": "^4.0.8",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.uniqby": "^4.7.0",
     "lodash.uniqueid": "^4.0.1",
-    "luxon": "^3.4.3",
-    "markdown-it-abbr": "^1.0.4",
-    "markdown-it-container": "^3.0.0",
-    "markdown-it-deflist": "^2.1.0",
-    "markdown-it-emoji": "^2.0.2",
-    "markdown-it-footnote": "^3.0.3",
+    "markdown-it": "^13.0.1",
+    "markdown-it-abbr": "^2.0.0",
+    "markdown-it-container": "^4.0.0",
+    "markdown-it-deflist": "^3.0.0",
+    "markdown-it-emoji": "^3.0.0",
+    "markdown-it-footnote": "^4.0.0",
     "markdown-it-html5-embed": "^0.3.3",
     "markdown-it-images-preview": "^1.0.1",
     "markdown-it-imsize": "^2.0.1",
-    "markdown-it-ins": "^3.0.1",
+    "markdown-it-ins": "^4.0.0",
     "markdown-it-katex-external": "^1.0.0",
-    "markdown-it-mark": "^3.0.1",
-    "markdown-it-sub": "^1.0.0",
-    "markdown-it-sup": "^1.0.0",
+    "markdown-it-mark": "^4.0.0",
+    "markdown-it-sub": "^2.0.0",
+    "markdown-it-sup": "^2.0.0",
     "markdown-it-task-lists": "^2.1.1",
     "md-editor-rt": "^4.7.0",
     "mixpanel-browser": "^2.47.0",
     "popper-max-size-modifier": "0.2.0",
-    "react": "^18.2.0",
+    "react": "^18.3.1",
     "react-beautiful-dnd": "^13.1.1",
-    "react-big-calendar": "^1.8.4",
-    "react-chartjs-2": "^5.2.0",
+    "react-big-calendar": "^1.18.0",
+    "react-chartjs-2": "^5.3.0",
     "react-color": "^2.19.3",
-    "react-datepicker": "^4.18.0",
-    "react-dom": "^18.2.0",
-    "react-error-boundary": "^4.0.11",
-    "react-hook-form": "^7.46.2",
-    "react-i18next": "^13.2.2",
+    "react-datepicker": "^4.25.0",
+    "react-dom": "^18.3.1",
+    "react-error-boundary": "^4.1.2",
+    "react-hook-form": "^7.54.2",
+    "react-i18next": "^13.5.0",
     "react-image-crop": "^10.1.8",
-    "react-player": "^2.13.0",
-    "react-redux": "^8.1.2",
-    "react-router-dom": "^6.16.0",
-    "react-scripts": "5.0.1",
-    "react-secure-storage": "^1.3.0",
+    "react-player": "^2.16.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.30.0",
+    "react-secure-storage": "^1.3.2",
     "react-transition-group": "^4.4.5",
     "react-virtualized-auto-sizer": "^1.0.26",
     "react-window": "^1.8.11",
-    "stream-browserify": "^3.0.0",
-    "typescript": "^4.9.5",
-    "uuid": "^9.0.1",
+    "typescript": "^5.8.2",
+    "uuid": "^11.1.0",
+    "vite-plugin-svgr": "^4.3.0",
     "web-vitals": "^3.5.0",
-    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz",
     "yup": "^1.3.1"
   },
   "scripts": {
-    "start": "react-app-rewired start",
-    "start:checkCycles": "react-app-rewired start -- checkCycles",
-    "build": "react-app-rewired build --project tsconfig.build.json",
+    "start": "vite",
+    "build": "tsc --project tsconfig.build.json && vite build",
```

Comment:
@sricharan-varanasi noted that we want to keep `tsc` because `vite` on its own does not perform type checking.